appengine/src/main/java/com/google/googleinterns/gscribe/models/MultipleChoiceQuestion.java

```diff
@@ -39,16 +38,4 @@ public void setOptions(List<String> options) {
         this.options = options;
     }
 
-    @Override
```

Comment:
If you remove this, it won't be possible to check if two MultipleChoiceQuestions are same along with options. The correct way of doing it with inheritance can be found here: https://stackoverflow.com/questions/2066917/overriding-equals-hashcode-in-sub-classes-considering-super-fields

Reply:
Added the equals and hashcode function.